lms/envs/common.py

```diff
@@ -1997,6 +1997,24 @@
     "reddit",
 ]
 
+# Set these to the correct values for your OAuth2/OpenID Connect provider
+JWT_AUDIENCE_KEY = 'replace-me'
+JWT_SECRET_KEY = 'replace-me'
+JWT_ISSUER = 'replace-me'
+SOCIAL_AUTH_EDX_OIDC_ID_TOKEN_DECRYPTION_KEY = JWT_SECRET_KEY
```

Comment:
`SOCIAL_AUTH_EDX_OIDC_ID_TOKEN_DECRYPTION_KEY` is never used and should be removed